### DISCUSSION TITLE
Alert email formatting issue on mobile devices

### DISCUSSION THREAD

**Discussion by hdsfgsgrlsgtw12** [ccfos/nightingale#1306] [Q&A]
你好，请问下，告警发送的同一个邮箱，电脑端接受邮件告警格式正常，如下：
![image](https://user-images.githubusercontent.com/45323722/205293438-6066ab0a-9ee4-44a7-be1e-01afd46799b5.png)
手机端展示告警邮件格式有问题：
![image](https://user-images.githubusercontent.com/45323722/205293729-be452b6c-cfde-4cde-8af4-c86d3a155fdf.png)
app和pc端是用的同一个模版嘛？如果手机端要显示和邮件一样格式的话，在哪改格式呢？我在mail.tpl文件加了换行样式了，没生效
![image](https://user-images.githubusercontent.com/45323722/205294350-b802cbf6-6e58-4135-9a80-5df4d7e4bdab.png)

**Comment by UlricQin** [comment]
都是 mail.tpl ，一个模板

**Reply by hdsfgsgrlsgtw12** [reply]
嗯嗯发现是同一个模版了 那手机端要是改格式和邮件一样的话 是改mail.tpl模版这个嘛？
为什么app和pc端展示的格式不一样呢？
